app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::SubscriptionsController < ApplicationController
+  def create
+    if subscription_params[:customer_id].present? && subscription_params[:tea_id].present?
```

Comment:
right now this allows one sad path to pass, I was having issues if I did have one customer or tea id included.  it was working like it should In postman but was giving errors locally due to subscription not being able to be created with ids that don't exist.